mak/COPY

```diff
@@ -438,4 +468,4 @@ COPY=\
 	$(IMPDIR)\core\thread\osthread.d \
 	$(IMPDIR)\core\thread\package.d \
 	\
-	$(IMPDIR)\etc\linux\memoryerror.d
+	$(IMPDIR)\etc\linux\memoryerror.d \
```

Comment:
This doesn't look correct?

Reply:
Thanks for noticing, fixed. (It had a missing newline escape and I apparently added where it wasn't necessary while trying to quickly amend my commit.)